qemu/tests/cfg/virtio_console.cfg

```diff
@@ -229,12 +229,12 @@
                             virtio_port_type_vs1 = serialport
                             virtio_port_params_vs1 = "nr=1"
                         - boot_too_much_ports:
-                            # arm doesn't use pci bus
-                            no aarch64
+                            # arm-mmio doesn't use pci bus
+                            no arm64-mmio
                             # max_ports extended in rhev7: "rhev7,lower_version"
                             max_ports_invalid = "512,32"
                             max_ports_valid = "511,31"
-                            extra_params = " -device virtio-serial-pci,max_ports=%s"
+                            extra_params = " -device %s,max_ports=%s"
```

Comment:
I am not sure if "virtio-serial-device" also extends max_ports in latest qemu-kvm-rhev.

Reply:
I don't really know, I tested this with `ccw` device on s390. Anyway I do get `max_ports=uint32` in `qemu-system-x86_64 -device virtio-serial-device,?` on `2.11.50 (v2.11.0-355-g281f327-dirty)`.

Reply:
Thanks for update. Actually `boot_too_much_ports` is not a serious case, I am wondering if we could keep testing `max_ports` in `pci` only this time.

Reply:
I think it make sense to use the same way of getting the device as it simplifies future changes (this is part of mass-update)

Reply:
Thanks, got it!